web/src/main/angular/src/app/core/components/server-map/class/merge-server-map-data.class.ts

```diff
@@ -132,3 +213,27 @@ function mergeHistogramByType(old: ILinkInfo, neo: ILinkInfo, histogramType: str
         old[histogramType] = neo[histogramType];
     }
 }
+function mergeResponseStatisticsByType(old: ILinkInfo, neo: ILinkInfo, srcOrTarget: string): void {
+    if (old[srcOrTarget]) {
```

Comment:
There is no possibility `neo` doesn't have `srcOrTarget` property? If there is, you may want to check it first

Reply:
@binDongKim  Updated and please recheck this again.